src/hooks/player_management/assign_physical_index.cpp

```diff
@@ -120,28 +120,12 @@ namespace big
 						{
 							dynamic_cast<player_command*>(command::get("breakup"_J))->call(plyr, {});
 						}
-						else
+						else if (g.protections.desync_kick)
```

Comment:
smart kick should probably be subbed in here so we can call that once and it will handle the host logic.